md/minimizer.py

```diff
@@ -36,11 +37,17 @@ def minimize_4d(romol, host_system, host_coords, ff, box):
     box: np.ndarray [3,3]
         Box matrix for periodic boundary conditions. units of nanometers.
 
+    Returns
+    -------
+    np.ndarray
+        This returns minimized host_coords.
+
     """
 
     host_bps, host_masses = openmm_deserializer.deserialize_system(host_system, cutoff=1.2)
 
-    ligand_masses = [a.GetMass() for a in romol.GetAtoms()]
+    # keep the ligand rigid
+    ligand_masses = [a.GetMass()*100000 for a in romol.GetAtoms()]
```

Comment:
Not something to address in this PR, but I'm noting that we frequently want to tell the integrator not to update certain coordinates, and perhaps in the new year we can brainstorm if there's a more foolproof way to do this.

Reply:
+1 to this. This was just a dirty hack.